landing-pages/site/content/en/blog/airflow-2.3.0/index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Apache Airflow 2.3.0 is here"
+linkTitle: "Apache Airflow 2.3.0 is here"
+author: "Ephraim Anierobi"
+github: "ephraimbuddy"
+linkedin: "ephraimanierobi"
+description: "We're proud to announce that Apache Airflow 2.3.0 has been released."
+tags: [Release]
+date: "TBD"
+---
+
+Apache Airflow 2.3.0 contains over TBD commits since 2.2.0 and includes TBD new features, TBD improvements, TBD bug fixes, and several doc changes.
+
+**Details**:
+
+📦 PyPI: https://pypi.org/project/apache-airflow/2.3.0/ \
+📚 Docs: https://airflow.apache.org/docs/apache-airflow/2.3.0/ \
+🛠️ Changelog: https://airflow.apache.org/docs/apache-airflow/2.3.0/changelog.html \
+🐳 Docker Image: docker pull apache/airflow:2.3.0 \
+🚏 Constraints: https://github.com/apache/airflow/tree/constraints-2.3.0
+
+As the changelog is quite large, the following are some notable new features that shipped in this release.
+
+## Dynamic Task Mapping(AIP-42)
+
+There's now first-class support for dynamic tasks in Airflow. What this means is that you can generate tasks dynamically at runtime. Much like using a `for` loop
+to create a list of tasks, here you can create the same tasks without having to know the exact number of tasks ahead of time.
+
+You can have a `task` generate the list to iterate over, which is not possible with a `for` loop.
+
+Here is an example:
+
+```python
+@task
+def make_list():
+    # This can also be from an API call, checking a database, -- almost anything you like, as long as the
+    # resulting list/dictionary can be stored in the current XCom backend.
+    return [1, 2, {"a": "b"}, "str"]
+
+
+@task
+def consumer(arg):
+    print(list(arg))
+
+
+with DAG(dag_id="dynamic-map", start_date=datetime(2022, 4, 2)) as dag:
+    consumer.expand(arg=make_list())
+```
+
+More information can be found here: [Dynamic Task Mapping](https://airflow.apache.org/docs/apache-airflow/latest/concepts/dynamic-task-mapping.html)
```

Comment:
```suggestion
More information can be found here: [Dynamic Task Mapping](https://airflow.apache.org/docs/apache-airflow/2.3.0/concepts/dynamic-task-mapping.html)
```

Probably better to link to the specific version? Or it at least should be to `stable`.